packages/docs/developer-resources/integrations/listings.md

```diff
@@ -0,0 +1,123 @@
+
+
+## Address for CELO and cUSD
+The smart contract address for CELO can be found [here]().
+The smart contract address for cUSD can be found [here]().
+
+
+## Useful API endpoints
+The following are useful API endpoints available to you that would help you in your listings of the CELO and cUSD digital assets.
+
+### CELO and cUSD
+For querying the API on total coins in circulation in CELO and cUSD, the following endpoint will provide you with that:
+```sh
+$ curl  [https://thecelo.com/api/v0.1.js?method=ex_totalcoins](https://thecelo.com/api/v0.1.js?method=ex_totalcoins) {"code":"200","msg":"success","data":{"CELO":608485841.9959723,"cUSD":10250632.56099673}}
+```
+
+### CP-DOTO 
+
+For API endpoints useful for listing that follow [CMC requirements](https://docs.google.com/document/d/1S4urpzUnO2t7DmS_1dc4EL4tgnnbTObPYXvDeBnukCg/edit#)
+
+#### Summary
+Summary overview.
+
+```sh 
+$ curl https://thecelo.com/api/v0.1.js?method=ex_summary
+
+{"trading_pairs":"CELO_CUSD","last_price":2.6143,"lowest_ask":2.5933609958506225,"highest_bid":2.5676,"base_volume":37524.32000000003,"quote_volume":14714.520000000002,"price_change_percent_24h":3.7027120070382127,"highest_price_24h":2.649,"lowest_price_24h":2.4787}}
+```
+
+#### Assets
+
+```sh
+$ curl https://thecelo.com/api/v0.1.js?method=ex_assets
+
+{"code":"200","msg":"success","data":{"CELO":{"name":"CELO","unified_cryptoasset_id":"5567","can_withdraw":"true","can_deposit":"true","min_withdraw":"0.000000000000000001","max_withdraw":"0.000000000000000001","maker_fee":"0.00","taker_fee":"0.005"},"CUSD":{"name":"Celo Dollars","unified_cryptoasset_id":"825","can_withdraw":"true","can_deposit":"true","min_withdraw":"0.000000000000000001","max_withdraw":"0.000000000000000001","maker_fee":"0.00","taker_fee":"0.005"}}}
+```
+
+#### Ticker
+
+```sh
+$ curl https://thecelo.com/api/v0.1.js?method=ex_ticker
+
+{"code":"200","msg":"success","data":{"CELO_CUSD":{"base_id":"5567","quote_id":"825","last_price":2.6124,"quote_volume":14789.520000000002,"base_volume":37720.30000000003,"isFrozen":"0"}}}
+```
+
+#### Orderbook
+
+```sh
+$ curl https://thecelo.com/api/v0.1.js?method=ex_orderbook
+
+{"code":"200","msg":"success","data":{"timestamp":1601061465962,"bids":[["2.5964","100"]],"asks":[["2.622606871230003","100"]]}}
+```
+
+#### CELO cUSD
+```sh 
+$ curl https://thecelo.com/api/v0.1.js?method=ex_celocusd
+
+{"code":"200","msg":"success","data":{"CELO_CUSD":[{"trade_id":2697341,"timestamp":1601061491,"price":0.38238291620515147,"quote_volume":25,"base_volume":65.37948987916423,"type":"Sell"},{"trade_id":2697336,"timestamp":1601061466,"price":0.382293821845672,"quote_volume":25,"base_volume":65.39472670341044,"type":"Sell"}]}}
+```
+
+
+## Stability Protocol
+
+To learn more about the Stability Mechanism, you can find it over [here](https://docs.celo.org/celo-codebase/protocol/stability). 
```

Comment:
worth linking to https://celoreserve.org ?

Reply:
added!